src/launchpad/size/analyzers/android.py

```diff
@@ -243,14 +245,3 @@ def _get_hermes_reports(self, apks: list[APK]) -> dict[str, HermesReport]:
                     logger.warning(f"Duplicate Hermes report key found: {relative_path}, overwriting")
                 all_reports[relative_path] = report
         return all_reports
-
-    def _get_images(self, apks: list[APK], file_analysis: FileAnalysis) -> dict[Path, FileInfo]:
-        image_map = {}
-        for apk in apks:
-            for image_file in apk.get_images():
-                image_map[image_file] = next(
-                    (file_info for file_info in file_analysis.files if str(image_file).endswith(file_info.path)), None
```

Comment:
This change should also improve performance as having to iterate through all the files would have been very slow.